apps/frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -26,11 +41,15 @@ const ProtectedRoute = ({ children }: Properties): JSX.Element => {
 		);
 	}
 
-	return hasAuthenticatedUser ? (
-		<>{children}</>
-	) : (
-		<Navigate replace to={AppRoute.SIGN_IN} />
-	);
+	if (!hasRequiredPermission) {
+		return <Navigate replace to={AppRoute.ANY} />;
```

Comment:
If I get it right when user doesn't have the required permission (status 403 forbidden) then the user is redirected to ANY route which displays not found page with status 404.

Reply:
Yes, that's right. I don't know what better error to return, since 403 is correct because access is forbidden. I can change to 404 not Found, but I'm not sure that this is a good decision